.github/workflows/deploy.yml

```
@@ -43,13 +42,15 @@ jobs:
           echo $NEW_VERSION > version.txt
 
-      - name: Build and push frontend image
+      - name: Build and push multi-architecture image
         run: |
-          docker build -t mdv-frontend:${{ env.NEW_VERSION }} -f Dockerfile .
-          docker tag mdv-frontend:${{ env.NEW_VERSION }} ${{ secrets.DOCKER_HUB_USERNAME }}/mdv-frontend:${{ env.NEW_VERSION }}
-          docker tag mdv-frontend:${{ env.NEW_VERSION }} ${{ secrets.DOCKER_HUB_USERNAME }}/mdv-frontend:latest
-          docker push ${{ secrets.DOCKER_HUB_USERNAME }}/mdv-frontend:${{ env.NEW_VERSION }}
-          docker push ${{ secrets.DOCKER_HUB_USERNAME }}/mdv-frontend:latest
+          echo "Building and pushing multi-architecture image for linux/amd64 and linux/arm64..."
+          
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)